tests/acceptance/features/bootstrap/CardDavContext.php

```diff
@@ -155,14 +155,11 @@ public function theAdministratorHasSuccessfullyCreatedAnAddressBookNamed($addres
 	 * @throws \Exception
 	 */
 	public function theCardDavHttpStatusCodeShouldBe($code) {
-		if ((int) $code !== $this->response->getStatusCode()) {
-			throw new \Exception(
-				\sprintf(
-					'Expected %s got %s',
-					(int) $code,
-					$this->response->getStatusCode()
-				)
-			);
-		}
+		$actualStatusCode = $this->response->getStatusCode();
+		Assert::assertEquals(
+			(int) $code,
+			$actualStatusCode,
+			"Expected: HTTP status code to be {$code } but got {$actualStatusCode}"
```

Comment:
```suggestion
			"Expected: HTTP status code to be {$code} but got {$actualStatusCode}"
```
I ma not sure if PHP will explode with a space after the variable name.

Reply:
fixed